pkg/util/env/env.go

```diff
@@ -69,6 +69,10 @@ func GetMustGatherTag() string {
 	return getenv("MUST_GATHER_TAG", "2.4")
 }
 
+func MustGatherEnable() bool {
```

Comment:
A bit confusing naming. IsMustGatherEnabled would be better.

Reply:
Done